src/ORM/Table.php

```diff
@@ -1436,6 +1435,36 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * Handles the logic executing of a worker inside a transaction.
+     *
+     * @param bool $atomic Whether to execute the worker inside a database transaction.
+     * @param callable $worker The worker that will run inside the transaction.
+     * @return mixed
+     */
+    protected function _executeTransaction($atomic, callable $worker)
```

Comment:
@ADmad it's not intended to be a replacement for `Connection::transactional`. So this method requires there to be a boolean argument passed. Otherwise it's a useless method, but I'll make the change anyway. It doesn't really change anything.